virtualfish/environment.fish

```diff
@@ -1,6 +1,30 @@
+# Set up the configurable filename
+if not set -q VIRTUALFISH_ENVIRONMENT_FILE
+    set -g VIRTUALFISH_ENVIRONMENT_FILE .env
+end
+
+function __vfsupport_set_env_file_path --description "Set VIRTUALFISH_ENVIRONMENT_FILE_PATH to appropriate value"
+    set -l vf_base (basename $VIRTUAL_ENV)
+    set -l vf_path $VIRTUAL_ENV/$VIRTUALFISH_ENVIRONMENT_FILE
+    # Check if projects plugin is used
+    if set -q PROJECT_HOME
+        set -l project_path $PROJECT_HOME/$vf_base/$VIRTUALFISH_ENVIRONMENT_FILE
+        if not test -r $project_path
+            and test -r $vf_path
+            set -g VIRTUALFISH_ENVIRONMENT_FILE_PATH $vf_path
+        else
+            set -g VIRTUALFISH_ENVIRONMENT_FILE_PATH $project_path
+        end
+    else
+        set -g VIRTUALFISH_ENVIRONMENT_FILE_PATH $vf_path
+    end
+end
```

Comment:
We can probably use `echo` instead of setting a global variable. I also realized the `basename` call doesn't need to be made if `$PROJECT_HOME` is not defined, so that should probably be moved further below. I also (very slightly) simplified the environment file path determination logic: 
```
function __vfsupport_env_file_path --description "Return full path to environment file"
    set -l vf_env_file_path $VIRTUAL_ENV/$VIRTUALFISH_ENVIRONMENT_FILE
    # Check if Projects plugin is used
    if set -q PROJECT_HOME
        set -l vf_name (basename $VIRTUAL_ENV)
        set -l proj_env $PROJECT_HOME/$vf_name/$VIRTUALFISH_ENVIRONMENT_FILE
        if test -r $proj_env
            echo $proj_env
        else
            echo $vf_env_file_path
        end
    else
        echo $vf_env_file_path
    end
end
```